azkaban-common/src/main/java/azkaban/executor/ExecutableFlow.java

```diff
@@ -64,6 +67,8 @@ public class ExecutableFlow extends ExecutableFlowBase {
   public static final String VERSIONSET_ID_PARAM = "versionSetId";
   private static final String PARAM_OVERRIDE = "param.override.";
   private static final String PROJECT_FILE_UPLOAD_USER = "uploadUser";
+  private static final String USER_DEFINED_RETRY_COUNT_PARAM = "userDefinedRetryCount";
+  private static final String SYSTEM_DEFINED_RETRY_COUNT_PARAM = "systemDefinedRetryCount";
```

Comment:
nit for subsequent PR: consider renaming to 
USER_DEFINED_**FLOW**_RETRY_COUNT_PARAM 
and
SYSTEM_DEFINED_**FLOW**_RETRY_COUNT_PARAM